docs/api.rst

```diff
@@ -8,8 +8,8 @@ API Documentation
 
 Here we list the Application Programming Interface (API) for pyprep.
 
-The :class:`NoisyChannels` class
---------------------------------
+The :class:`~pyprep.NoisyChannels` class
```

Comment:
cool, you solved the linking issue!

Why is the `~` tilde needed though?

Reply:
The tilde makes it so it only prints the name of the class/function/module/etc. instead of the whole path, i.e., "NoisyChannels" instead of "pyprep.NoisyChannels". A handy piece of .rst I picked up during the solving process!

Reply:
nice, thanks 👍